test/deprecated/IO/channel-writebits.chpl

```diff
@@ -0,0 +1,13 @@
+use FileSystem;
+use IO;
+
+config const testfile="test.bin";
+var f = open(testfile, ioMode.cwr);
+var r = f.writer(kind=ionative);
+var tmp = 0b101;
+
+// should throw deprecation
+r.writebits(tmp, 3);
+
+FileSystem.remove(testfile);
```

Comment:
This test is good as is, just an FYI: you can also set this up with `channel-writebits.catfiles` and `channel-writebits.cleanfiles` files, so that the output of `r.writebits` is tested against the good file and `test.bin` is removed after the test runs. See the `fileWriterHigh` test in the same directory for an example.

I wouldn't go through the trouble of changing it, just pointing this out for future reference.